magento/pwa-studio
# [Doc] Update out of date content in the custom footer tutorial

## Description

Update the Custom Footer tutorial to bring code samples up to date with recent footer changes.

## Related Issue

Closes #2623 
Closes PWA-863

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Navigate to the pwa-devdocs project: `cd pwa-devdocs`
1. Run the HTML preview server: `yarn develop`
1. Navigate to `/tutorials/pwa-studio-fundamentals/modify-site-footer/`
1. Verify content is up to date and accurate.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
